.github/DOCUMENTATION_TEMPLATE.md

```diff
@@ -0,0 +1,64 @@
+# How to conttribute to MFTF docs
+
+MFTF documentation is kept within the /docs/ folder in this repo.
+We welcome contributions to the documentation.
+This page describes the process for submitting docs and serves as a template so that docs are written properly.
+
+The contribution workflow for docs is the same as submitting code.
+To contribute to MFTF docs:
+
+1. Make a branch from the [MFTF repo][].
+1. Make edits/additions/deletions as needed.
+1. Submit your PR to the `develop` branch.
+
+Once submitted, it will be reviewed by a member of the documentation team.
+If approved it will be tested and merged.
+If it needs any work, we will inform you.
+
+Any changes to the Table of Contents will need to be made in the regular [devdocs repo][].
```

Comment:
Going to leave it in.

Reply:
Then you should mention that the documentation is also published on DevDocs website, because it is not clear what Table of Contents do you mean. And that to keep the table of contents consistent, the contributor will need to add the corresponding changes to the devdocs repository.

Reply:
Explained and linked to the TOC.

Reply:
I don't see the changes.